src/main/java/com/github/vfss3/S3FileProvider.java

```diff
@@ -28,6 +28,11 @@
  * @author Moritz Siuts
  */
 public class S3FileProvider extends AbstractOriginatingFileProvider {
+    /**
+     * Supported schema
+     */
+    public static final String SCHEMA = "s3";
```

Comment:
I meant there two places
com.github.vfss3.S3FileSystemConfigBuilder#S3FileSystemConfigBuilder
com.intridea.io.vfs.provider.s3.S3FileSystemConfigBuilder#S3FileSystemConfigBuilder
It is definitely not related to provider and schema. Maybe CONFIG_PREFIX or just PREFIX

Reply:
will be renamed to PREFIX